sunshine-testng/src/main/java/io/github/tatools/sunshine/testng/SunshineTestNG.java

```diff
@@ -1,5 +1,6 @@
-package io.github.tatools.sunshine;
+package io.github.tatools.sunshine.testng;
 
+import io.github.tatools.sunshine.core.*;
```

Comment:
Do we really need all those classes from `io.github.tatools.sunshine.core.*` here?